applications/IgaApplication/custom_modelers/refinement_modeler.cpp

```diff
@@ -172,6 +172,88 @@ namespace Kratos
                             << " by inserting knots in v, however insert_nb_per_span_v is set to 0." << std::endl;
                     }
                 }
+                if (rParameters["parameters"].Has("increase_degree_u")) {
```

Comment:
I think the whole increase polynomial degree section should be before the insert knots. I believe this way is more efficient.

Reply:
Totally agree. We'll go with p-h-refinement (k-refinement) this way, which leads to smoother continuity.